### PR TITLE
danger: help users to write server tests

### PR DESCRIPTION
**Pull request by niccokunzmann**
- fixes https://github.com/badges/shields/issues/1968
- adds the desired notice

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -65,7 +65,9 @@ if (packageJson.modified && !packageLock.modified) {
 if (server.modified && !serviceTests.createdOrModified) {
   warn([
     'This PR modified the server but none of the service tests. ',
-    "That's okay so long as it's refactoring existing code.",
+    "That's okay so long as it's refactoring existing code. ",
+    "Otherwise, please consider adding tests to the service: ",
+    "https://github.com/badges/shields/blob/master/doc/service-tests.md#readme",
```
Comment (platan) [inline dangerfile.js]: Do we need `#readme` at the of the url?

Comment (niccokunzmann) [reply dangerfile.js]: It skips directly to the things you like to read. The other documentation files are not interesting. I added it for ease of use. Why would you remove it?

Comment (platan) [reply dangerfile.js]: @niccokunzmann Thanks for explanation. I didn't notice the difference earlier. I agree with you, it's useful.

**Comment by platan** [comment]
@niccokunzmann thanks for your contribution!

**Pull request merged**